### PR TITLE
[#5] Messages/Alerts

### LINKED ISSUE #5: Messages / Alerts
Implement styles for messages / alerts. See design style guide in Figma.

Reference:
messages_alerts

### PR DESCRIPTION
**Pull request by LavanyaThuyamani**
Implements styles for messages / alerts with the help of figma design style guide.

### CODE REVIEW

File: css/src/_m-message.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
+.m-messages {
+    padding: 0;
+    font-style: italic;
+    list-style: none;
+
+    &__item {
+        --message-txt: #da394c;
+        --message-bg: rgba(238, 162, 171, 0.25);
+        position: relative;
```
Comment (mageshravi) [inline css/src/_m-message.scss]: Add empty line between css variables and properties.

File: css/src/_m-message.scss [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        width: fit-content;
+        margin-inline-end: 11px;
+        margin-block-end: 20px;
+        padding: 10px 15px;
+
```
Comment (mageshravi) [inline css/src/_m-message.scss]: Remove this empty line.

File: css/src/_m-message.scss [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    &__close-btn {
+        position: absolute;
+        top: -11px;
+        right: -11px;
+    }
+}
+
+@media (prefers-color-scheme: dark) {
```
Comment (mageshravi) [inline css/src/_m-message.scss]: Move this block inside the existing `.m-messages` block like shown below,

```scss
.m-messages {
    /* ... */

    @media (prefers-color-scheme: dark) {
        &__item {
            /* ... */
        }
    }
}

File: index.html [hunk 4]
```diff
@@ -74,6 +74,62 @@ <h6>Heading 6</h6>
...
+            <span class="m-messages__desc">This is a success message</span>
+        </li>
+    </ul>
+
+    <ul class="m-messages">
+        <li class="m-messages__item">
+            <img class="m-messages__close-btn" src="close-btn.svg" alt="close" />
+
+            <span class="m-messages__desc">This is an error message
+                that flows two lines full.</span>
```
Comment (mageshravi) [inline index.html]: Remove line break in this line.

File: index.html [hunk 5]
```diff
@@ -74,6 +74,62 @@ <h6>Heading 6</h6>
...
+        </li>
+
+        <li class="m-messages__item m-messages__item--warning">
+            <img class="m-messages__close-btn" src="close-btn.svg" alt="close" />
+
+            <span class="m-messages__desc">This is a warning
+                that flows two lines full.</span>
```
Comment (mageshravi) [inline index.html]: Remove line break.

File: index.html [hunk 6]
```diff
@@ -74,6 +74,62 @@ <h6>Heading 6</h6>
...
+        </li>
+
+        <li class="m-messages__item m-messages__item--info">
+            <img class="m-messages__close-btn" src="close-btn.svg" alt="close" />
+
+            <span class="m-messages__desc">This is an information
+                that flows two lines full.</span>
```
Comment (mageshravi) [inline index.html]: Remove line break.

**Review by mageshravi — CHANGES_REQUESTED** [review]
Changes requested

File: css/src/_m-message.scss (hunk 1 above)
Comment (LavanyaThuyamani) [reply css/src/_m-message.scss]: Added an empty line between css variables and properties.

File: css/src/_m-message.scss (hunk 2 above)
Comment (LavanyaThuyamani) [reply css/src/_m-message.scss]: Removed the empty line in the 14th line.

File: css/src/_m-message.scss (hunk 3 above)
Comment (LavanyaThuyamani) [reply css/src/_m-message.scss]: Moved the @media block inside the existing `.m-messages` block like the above shown structure.

File: index.html (hunk 4 above)
Comment (LavanyaThuyamani) [reply index.html]: Removed the line break in this line.

File: index.html (hunk 6 above)
Comment (LavanyaThuyamani) [reply index.html]: Removed the line break in this line.

File: index.html (hunk 5 above)
Comment (LavanyaThuyamani) [reply index.html]: Removed the line break in this line.

**Review by mageshravi — APPROVED** [review]

**Pull request merged**